notebooks/03-query/11-joins.ipynb

```diff
@@ -0,0 +1,2475 @@
+   "metadata": {},
+   "source": [
+    "### Load `clicks` and `users` Into Separate Tables\n",
+    "\n",
+    "Load separate tables for `clicks` and `users` to demonstrate join funtionality."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a819cf3-1403-42d4-81de-fbf2991c6c9e",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "#### Generate Ingestion SQL from Data Files\n",
+    "While this notebook already includes the ingestion SQL, here are the details on how to generate it using the [Druid Console](http://localhost:8888/unified-console.html#):"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5443904f-6abb-4fb9-aec3-39d564de9f06",
+   "metadata": {},
+   "source": [
+    "Instead of manually figuring out the EXTERN table syntax needed to build the ingestion SQL statement, you can generate the SQL statements from the data file by using the Druid Console and following these steps:\n",
+    "\n",
+    "1. Click on `Load` and select `Batch-SQL`\n",
+    "   <img src=\"assets/load-data-batch.png\" alt=\"drawing\" style=\"width:400px;\"/>\n",
+    "3. Select an HTTP source and type in the URL to one of the generated file, the data generator service makes them avaialable at:\n",
+    "   - `http://datagen:9999/file/users.json`\n",
+    "   - `http://datagen:9999/file/clicks.json`\n",
+    "4. Click on `Connect Data` and Druid Console will parse and present a few rows of data from the file.\n",
+    "\n",
+    "   ![](assets/load-get-sql.png)\n",
+    "5. Click on `Skip the wizard and continue with custom SQL`, the SQL is displayed in a new Query view tab ready to edit or run."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6beb309b-2a1a-43c7-a001-86b61e47adf4",
+   "metadata": {},
+   "source": [
+    "### Execute the Batch Ingestions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "2f1d9978-ff4f-4b77-b926-b42ce17419a3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Loading data, status:[SUCCESS]: 100%|██████████| 100.0/100.0 [00:09<00:00, 10.77it/s]\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2023-10-26 00:58:42.329707 <<<<<<<<<<<<< \n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Loading data, status:[SUCCESS]: 100%|██████████| 100.0/100.0 [00:30<00:00,  3.28it/s]            \n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "2023-10-26 00:59:12.946475 <<<<<<<<<<<<< \n",
+      "2023-10-26 00:59:16.892032 <<<<<<<<<<<<< \n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "datetime.datetime(2023, 10, 26, 0, 59, 16, 892032)"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```

Comment:
In this section, you will execute two ingestions:

1) Does beans
2) Does sausages

Run the following cell to ingest beans.

BLAH

Here's what beans look like:
SELECT

Run this cell to ingest sausages.
BLAH
THis is what sausages look like.

Reply:
done.